index.html

```diff
@@ -372,6 +376,7 @@
       1. <a data-cite="!DOM#concept-event-fire">Fire an event</a> named `change` at the `NetworkInformation` object.
 </section>
 
+<section>
```

Comment:
It was a red herring.. I noticed that the wicg version mangled the privacy section and tried wrapping it in section headers, but when I backed it out and hard-refreshed the problem went away.. it must have been a transient issue earlier. That said, decided to keep the section wrappers, since all the other sections do.